### ISSUE TITLE
Enhancement request:  Per channel polling frequency

### ISSUE THREAD

**Issue by jasonkgreen** [jackjamieson2/yarns-microsub-server#97]
Some feeds, like social media, are busy enough to make frequent polling desirable. Others, like blog feeds , could be polled less frequently.  This might also reduce overhead since not all channels would be polled each time

**Comment by jackjamieson2** [comment]
Thanks for this suggestion.

Yarns already has a feature that accomplishes some of this in the background, though you just made me realize it's not documented or visible to users, which is not ideal

Basically, Yarns automatically updates its polling frequency on a per-feed level. Feeds that regularly have new content are polled more often, and feeds with less frequent updates are polled less frequently. 

Does that accomplish more or less what you're thinking of? 

I think it would be nice to actually indicate this the admin screen somewhere since it just happens invisibly in the background

**Comment by jasonkgreen** [comment]
Yes, that's exactly the kind of thing I was thinking of. I just didn't know it was happening.  Thanks for your prompt reply.


________________________________

**Comment by jackjamieson2** [comment]
Great! I'll leave this issue open as a reminder to think about how to best convey this to users (through documentation, UI, or both)
